content/posts/android-tv-14.md

```diff
@@ -0,0 +1,110 @@
+
+- Fixed the incorrect sizing of channel images in some places
+- Fixed an issue where the guide could have some sizing problems with many channels
+- Fixed the "null" text when information was missing in a some places
+- Fixed guide sorting not always working
+- Removed the category images causing contrast issues
+- Restyled some buttons to use theme colors
+- Restyled the filter and options popups to the same design as other parts of the app
+
+## Improved browsing grid
+
+To make it easier to see a lot of items or just a few we have added two new image sizes in the grid view: smallest and extra large. Together with this change the sizing is now based on the actual size of your device, fixing some bugs on 4k displays. This might change the size you're used to a bit. We also fixed some issues with the vertical direction and now consider it a stable feature!
+
+## New crash reporting
+
+Back in 2020 with version 0.11 we added a crash reporting utility. This function helped us fix a lot of issues with the app. The crash reporting relied on a third-party service to collect the reports. We kept tight control of who could access this data, but never liked this approach as it did use a third-party service we have no control over. In this update the crash reporting code was rewritten and now sends all reports to your own Jellyfin server. The crash reports use a new format that makes it more readable for both the user and developers and includes details to identify and fix bugs even faster.
+
+Crash reporting is now turned on by default. When the app crashes it will send a report to the latest server you used. On fresh installations the report is queued until a server is connected to.
+
+## Markdown support for item & server descriptions
+
+The support for basic HTML elements in item descriptions is extended to now also support Markdown. The same HTML and Markdown support is also added to server descriptions. With this change you can easily use **bold**, *italic* or ~~strikethrough~~ text.
+
+## Rewrite of WebSocket code
+
+We've seen a lot of issues with WebSockets in the past. This was partially caused by outdated code in the app. This part of the app is now rewritten to use an up-to-date library via our own SDK. In our testing the connections are more stable and it allows us to add more code that relies on this connection in the future, like instant updates of items on the home screen and SyncPlay.
+
+Visit our [networking](https://jellyfin.org/docs/general/networking/index.html) documentation if you're using a reverse proxy and still have connection issues.
+
+## Contributors
+
+Jellyfin apps are developed by great contributors, we couldn't do it without them. Everyone is a volunteer that doesn't get paid by organizations with self-interest, consider donating if you appreciate their work. A big shout-out to all contributors that made this release possible:
```

Comment:
The "self-interest" part is of course true but disturbs the reading flow a bit.
Maybe "any organizations (which usually act out of self-interest), ..." using parentheses?

```suggestion
Jellyfin apps are developed by our great contributors and we couldn't do it without them. Everyone is a volunteer that doesn't get paid by any organizations, so consider donating if you appreciate their work. A big shout-out to all contributors that made this release possible:
```